pkg/kubeapiserver/server/insecure_handler.go

```diff
@@ -20,14 +20,20 @@ import (
 	"net/http"
 
 	genericapifilters "k8s.io/apiserver/pkg/endpoints/filters"
+	"k8s.io/apiserver/pkg/features"
 	"k8s.io/apiserver/pkg/server"
 	genericfilters "k8s.io/apiserver/pkg/server/filters"
+	"k8s.io/apiserver/pkg/util/feature"
 )
 
 // BuildInsecureHandlerChain sets up the server to listen to http. Should be removed.
 func BuildInsecureHandlerChain(apiHandler http.Handler, c *server.Config) http.Handler {
 	handler := apiHandler
-	handler = genericfilters.WithMaxInFlightLimit(handler, c.MaxRequestsInFlight, c.MaxMutatingRequestsInFlight, c.LongRunningFunc)
+	if feature.DefaultFeatureGate.Enabled(features.APIPriorityAndFairness) && false {
```

Comment:
it doesnt have to be protected by the feature-gate, checking if it's nil would be sufficient. (see my last comment on where the flowcontrol system is constructed)

Reply:
I chose to not check `nil` on the grounds that if the admin asked for this feature and the field is `nil` then something is wrong and the apiserver should crash.

Reply:
the `&& false` deserves a comment?

Reply:
Definitely.
It is a temporary thing I introduced for debugging during initial development.  It was motivated by a deadlock inside the code, which prevented all future requests from being handled --- including calls to GET /debug/pprof/ things.

Reply:
Added the comment.